it/test/uk/gov/hmrc/alcoholdutycalculator/RatesIntegrationSpec.scala

```diff
@@ -55,4 +66,188 @@ class RatesIntegrationSpec extends ISpecBase {
       rateBand.rateType mustBe RateType.Core
     }
   }
+
+  "validate rate file against schema" in {
+    val stream: InputStream = getClass.getResourceAsStream("/rates/alcohol-duty-rates.json")
+    val lines               = scala.io.Source.fromInputStream(stream).getLines().mkString
+    validateJsonAgainstSchema(lines) mustBe true
+  }
+
+  "validate the rates within the file" - {
+    // For every tax year we need a section! Seq(years and loop) up to a max of cy-5
+    s"For cy" - {
```

Comment:
Rate Periods are not stored per tax or calendar year. Sometimes there can be multiple rate period within the year if there are multiple budgets due to government change or they can stay the same for more than 1 year. That is why we called them rate period so we should follow the same naming